uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/document.js

```diff
@@ -1,4 +1,4 @@
-/* globals angular */
+/* globals angular, _ */
```

Comment:
sorry, just caught this. `angular` and `_` cannot be global variables. They come from the `common/angular` and `common/lodash` module, so just inject those